Automattic/vip-go-mu-plugins
# e2e tests: Initial framework, tests, and CI integration

## Description

This PR adds the Playwright test framework that will be used for end to end testing of mu plugins. Six new tests and configuration to run the tests in CI have also been added.

The tests will run against any WP environment, but I've added scripts to start a default test environment using the VIP cli.



## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test


1. Follow the instructions on the readme to start test environment and run tests.
2. Verify tests run and pass in CI (Github actions)
